server.js

```diff
@@ -0,0 +1,30 @@
+const http = require('http');
+const app = require('./lib/app');
+
+http.createServer(app)
+  .listen(7890);
```

Comment:
```suggestion
  .listen(PORT, () => console.log(`Listening on ${PORT}`));
```
Sometimes helpful to have confirmation that your server is up and running.